src/webgl/framebuffer.js

```diff
@@ -8,6 +8,24 @@ import {getTypedArrayFromGLType, getGLTypeFromTypedArray} from '../utils/typed-a
 import {log} from '../utils';
 import assert from 'assert';
 
+// Local constants - will collapse during minification
+const GL_FRAMEBUFFER = 0x8D40;
```

Comment:
> Why we decide to have these GL constants defined here instead of extracting them from our "GL" struct?

The comment is a little unclear, sorry. It's related to making the libraries smaller. The GL object contains all flags and is pretty big (15Kb+). If we have local constants here, they collapse to small numbers during minification/uglification. Then the app can decide whether to import GL or not. Small apps won't have to take the hit.